scilpy/tracking/tools.py

```diff
@@ -1,7 +1,139 @@
 # -*- coding: utf-8 -*-
```

Comment:
About the name tracking/tools vs tracking/utils. Is the difference between the two clear enough for us, or could we find a better name than tools? I have no opinion.

Reply:
It is unclear. We don't use `module/tools.py` elsewhere besides `tractanalysis`, and even there it only has two functions.

IMO the content of tools.py could go into utils

Reply:
Go for it, I think. @arnaudbore ?

Reply:
I transferred everything from `tracking.tools` to `tracking.utils` in 54d82b8